hack/e2e-suite/services.sh

```diff
@@ -388,8 +388,13 @@ verify_from_container "${svc3_name}" "${svc3_ip}" "${svc3_port}" \
 #
 echo "Test 5: Remove the iptables rules, make sure they come back."
 echo "Manually removing iptables rules"
-ssh-to-node "${test_node}" "sudo iptables -t nat -F KUBE-PORTALS-HOST"
-ssh-to-node "${test_node}" "sudo iptables -t nat -F KUBE-PORTALS-CONTAINER"
+server_version=$(kube_server_version)
+if [[ ${server_version} -ge 800 ]]; then
```

Comment:
This actually has little to do with the server version and everything to do with the kubelet version.